packages/neo-one-client-common/src/crypto.ts

```diff
@@ -575,6 +586,192 @@ const isSignatureContract = (script: Buffer) => script.length === 35 && script[0
 
 const isStandardContract = (script: Buffer) => isSignatureContract(script) || isMultiSigContract(script);
 
+const HARDENED_KEY_OFFSET = 0x80000000;
```

Comment:
Can you add references to the documentation for all of the constants in this PR (similar to below).